main.c

```diff
@@ -1035,9 +1037,12 @@ static void kbd_key(void *data, struct wl_keyboard *k, uint32_t serial,
 		b = b->next;
 	}
 
-	if (!action)
-		tsm_vte_handle_keyboard(term.vte, sym, XKB_KEY_NoSymbol,
+	if (!action){
+		bool vte_write_success = tsm_vte_handle_keyboard(term.vte, sym, XKB_KEY_NoSymbol,
```

Comment:
This variable isn't needed, just put the call in the `if`

Reply:
I'm also curious, why only scroll down if it returns true here? Is this the behaviour of other terminals?